lib/assets/javascripts/kit.js

```diff
@@ -0,0 +1,18 @@
+function ready (fn) {
```

Comment:
Would it be OK to scope this and make it more descriptive?  I'd consider something like:

`PrototypeKit.documentReady`

Reply:
Sure. How about `GOVUKPrototypeKit.documentReady`, to be consistent with `GOVUKFrontend`?

We might also want to consider attaching the `GOVUKPrototypeKit` object to `window`, again to follow the Design System's convention.

Reply:
Raised in a separate PR https://github.com/alphagov/govuk-prototype-kit/pull/1490 as this PR still needs work.

Reply:
Fixed in main